src/ontology/cl-edit.owl

```diff
@@ -19317,14 +19317,16 @@ AnnotationAssertion(rdfs:label obo:CL_0002575 "central nervous system pericyte"^
 EquivalentClasses(obo:CL_0002575 ObjectIntersectionOf(obo:CL_0000669 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001017)))
 SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_0002575 obo:CL_0000669)
 
-# Class: obo:CL_0002576 (perineural cell)
+# Class: obo:CL_0002576 (perineurial cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GO:tfm"^^xsd:string) Annotation(oboInOwl:hasDbXref "http://en.wikipedia.org/wiki/Perineurium"^^xsd:string) obo:IAO_0000115 obo:CL_0002576 "A myofibroblast that lies in the connective tissue of the spinal cord that has a distinctly lamellar arrangement."^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GO:tfm"^^xsd:string) Annotation(oboInOwl:hasDbXref "NCIT:C41442"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:25818566"^^xsd:string) Annotation(oboInOwl:hasDbXref "http://en.wikipedia.org/wiki/Perineurium"^^xsd:string) Annotation(rdfs:comment <https://orcid.org/0000-0001-7655-4833>) obo:IAO_0000115 obo:CL_0002576 "A glial cell that is part of the perineurium. This cell type has thin long bipolar cytoplasmic processes, pinocytotic vesicles, fragments of external lamina and/or external lamina-like material, attachment plaques, and desmosome-like junctions."^^xsd:string)
```

Comment:
Consider adding that perineurial cells (unlike fibroblasts) have double basement membranes (os basal lamina).

Note: Probably more for perineurium, but an important function of perineurial cells is being "a component of the blood–nerve barrier, protecting the interior of the nerve from toxins, infection, and ionic flux" and are connected via tight junctions.

Reply:
Thanks for the feedback, @aleixpuigb.

1- Can you provide a ref to support the first statement?

2- Agreed. Would you kindly update perineurium in Uberon so it is part of the ['blood nerve barrier'](http://purl.obolibrary.org/obo/UBERON_0003209)? Then the latter term can be imported to CL. Ideally 'perineurial cell' will be inferred to be part of the 'blood nerve barrier' as well.

Reply:
1- It's from PMID:25818566
"Individual perineurial cells within this structure are extremely thin, have a double basal lamina, and are fitted together by overlapping border zones"
Additionally, perineurial cells have double basement membranes, whereas fibroblasts only have a single basal laminae
2- Ok, I will do that

Reply:
Content looks great, but spotted some grammatical issues with this text:

> Perineurial cells historically have been referred to as fibroblasts because of shape; however, unlike fibroblasts, a perineurial cell: does not have a compact nucleus and large endoplasmic reticulum; does have a double basement membrane opposed to a single basal lamina; are carefully joined by tight junctions into a single sheet as opposed to arranged in a large mass; and finally, can surround a small axon bundle at a nerve terminal whereas a fibroblast cannot."^^xsd:string)

"**_a_** perineurial cell... **_are_** carefully joined by tight junctions"